uid-pages/page-admin-profile-list/src/resources/page.properties

```diff
@@ -1,5 +1,5 @@
 displayName=Bonita Admin Profile List
 name=custompage_adminProfileListBonita
 description=This page provides the list of a profiles. It is dedicated to Admin-like profiles. You can export it and edit it in the UI Designer.
-resources=[]
+resources=[GET|system/feature, GET|portal/profile, POST|portal/profile, PUT|portal/profile, DELETE|portal/profile, GET|portal/profileMember, POST|portal/profileMember, DELETE|portal/profileMember, POST|services/profile/import, GET|identity/user, GET|identity/group, GET|identity/role]
```

Comment:
POST|services/profile/import is weird.
I don't think it can work when deploying this page since I cannot find it in bonitasoft/bonita-engine-sp/blob/release-7.12.0/community/platform/platform-resources/src/main/resources/tenant_template_portal/resources-permissions-mapping.properties
We probably need to review this URL in the page or change it to POST|profile/import